<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VkAudioSync::MainWindow</class>
 <widget class="QMainWindow" name="VkAudioSync::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>13</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VKAudioSync</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/application/share/icons/hicolor/64x64/apps/vkaudiosync.png</normaloff>:/application/share/icons/hicolor/64x64/apps/vkaudiosync.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QFrame" name="leftColumn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>165</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="userpic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap resource="../icons.qrc">:/application/share/icons/hicolor/64x64/apps/vkaudiosync.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="username">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="remember">
           <property name="text">
            <string>Remember</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="langList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>133</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string notr="true">English</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/famfam/icons/gb.png</normaloff>:/famfam/icons/gb.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Русский</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/famfam/icons/ru.png</normaloff>:/famfam/icons/ru.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Українська</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/famfam/icons/ua.png</normaloff>:/famfam/icons/ua.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Polska</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/famfam/icons/pl.png</normaloff>:/famfam/icons/pl.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Беларуская</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/famfam/icons/be.png</normaloff>:/famfam/icons/be.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Login...</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/famfam/icons/key_go.png</normaloff>:/famfam/icons/key_go.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Choose dir...</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/famfam/icons/folder_go.png</normaloff>:/famfam/icons/folder_go.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="syncButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Synchronize</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/famfam/icons/arrow_refresh.png</normaloff>:/famfam/icons/arrow_refresh.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="aboutButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>About...</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/famfam/icons/book.png</normaloff>:/famfam/icons/book.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="proxyButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Proxy</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/famfam/icons/proxy.png</normaloff>:/famfam/icons/proxy.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>7</height>
            </size>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/famfam/icons/door.png</normaloff>
             <activeon>:/famfam/icons/door_open.png</activeon>:/famfam/icons/door.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sort By</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sorter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>-</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Artist</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Duration</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Progress</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Status</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="AudioListView" name="listView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AudioListView</class>
   <extends>QListView</extends>
   <header>audiolistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
